verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -123,6 +124,14 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
         );
     }
 
+    scrollTo(name: string): void {
+        Scroll.scroller.scrollTo(name, {
+            duration: 100,
+            smooth: true,
+            offset: -64, // Account for header height
```

Comment:
Is this available as a reference (without too much sweat)? So that we don't goof/miss this if the header's changed. Nw if not.

Reply:
I'm quite sure the entire header + create new form page will be reconstructed to fit the mocks. So I'll leave for now but keep in mind for the final product.